ambry-commons/src/main/java/com.github.ambry.commons/ServerErrorCode.java

```diff
@@ -23,6 +23,7 @@
   No_Error,
   IO_Error,
   Blob_Not_Found,
+  Blob_Authorization_Failure,
```

Comment:
Could you move `Blob_Authorization_Failure` to the end? It's safe to append new error code to enum.